### DISCUSSION TITLE
Promoting pets to Gods

### DISCUSSION THREAD

**Discussion by drumdrumdrum3337** [sunflower-land/sunflower-land#4469] [Ideas]
Context: arguably faction pets are not a compelling part of the gameplay. They don’t promote team work and are mostly interesting to those collecting marks, it feels like XP boost is much less relevant. They don’t add much to the game lore and from that perspective feel like a dead end. 

But there is a solution! Due to some unforeseen mutations and overeating pets become Gods!

Having Gods in the game would mean that we, humble farmers, need to bring them sacrifices to make sure they are in good mood. 

Here’s an example how it might work:
- Every farmer decide themselves what to sacrifice each week and how much. 

- Gods are puny and capricious, they don’t obey to rules and math and streaks, but they will be grateful to those bringing sacrifices in their own way. 

> >Those who sacrifice a little, will get something small (e.g. +0.3 carrots for the upcoming week); those who give more will get something better (e.g. +0.2 iron, or -2hrs on baking cakes for the week); and those who sacrifice generously will be rewarded generously as well (e.g. +1 kale for the week). Very rarely Gods might even gift a collectible with a small permanent boost!  

- All gifts from Gods are random and individual for every player.

- There are servants to Gods that will also remunerate us with marks for bringing our sacrifices. 

This covers individual gameplay. Basically, the idea is that the gameplay becomes a bit more unpredictable (one never knows what Gods will give in return) but without the damaging rng, and it’s up to the player to decide how much to sacrifice. 

However, factions should participate as teams as well! Here’s how:

- Gods love when there are places of worship built to their name, so every faction can also embark on building such a place for their God that (as all places of such magnitude) will be crazy expensive and take very long time (months?) to be built. But it’s worth it as Gods will be even more generous then, especially to those who donated more. 

- Building the place of worship could be as simple as donating required resources weekly, or could be more complex and require certain actions on the building site (hauling stones, cutting wood, etc). 

That’s roughly it. This is indeed a very high level idea and further details need to be developed (mechanics, lore, etc) in case it would be considered worth exploring further.

**Comment by fra-vh** [comment]
As long as this is catered more towards small-medium players and not whales, this seems like an amazing idea.

- requirement sacrifice to receive a gift should be something like this:

*small -> equivalent to the resources of 1-2 ticket deliveries 
*medium -> equivalent to the resources of 3-5 ticket deliveries
*small -> equivalent to the resources of 5-7 ticket deliveries

- all sacrifices should give marks, but capped (you only get marks before receiving the last gift, after last gift sacrifice is not possible)

**Comment by adamhannigan** [comment]
This is awesome @drumdrumdrum3337 

We've been playing around with some ideas with the Guardians on this - some awesome inspiration in this post 👀 👏 

It is great timing, we've got a proof of concept in the works

**Reply by drumdrumdrum3337** [reply]
Hey @adamhannigan I am wondering how things are going with the guardians? I have no data, but I see no excitement around that part of the gameplay. Since we now have NFT pets, maybe it’s time to promote the faction pet to the god level? :) it might spruce up social farming as well

**Comment by Truenist** [comment]
Cool idea, you could also come up with synergy with nft pets, like if you receive a blessing, a similar nft pet gives a multiplier if there is a combination. For example, a faction pet gives +0.5, and a boar additionally = +0.2 to acorns or +0.2 to cabbage.
